### PR TITLE
Add Bert SQUAD Interpret Part2

### PR DESCRIPTION
**Pull request by NarineK**
Adding Bert SQUAD interpret part 2 based on some literature review.

1. Attention is Not Only a Weight: Analyzing Transformers with Vector Norms
    https://arxiv.org/pdf/2004.10102.pdf
2. Revealing the Dark Secrets of BERT
    https://www.aclweb.org/anthology/D19-1445.pdf
3. What does BERT look at? An Analysis of BERT’s Attention
    https://www.aclweb.org/anthology/W19-4828.pdf

I'll add the description of the tutorial in a separate PR

### CODE REVIEW

**Review by vivekmig — APPROVED** [review]
Looks great @NarineK :) ! This is an awesome, really detailed tutorial with a lot of interesting analysis! Just a few minor comments:

1. I tried running the tutorial and had some errors, I believe it's related to the latest transformer version. With transformers 4.3.3 I received this error when running the predict in the first section TypeError: argmax(): argument 'input' (position 1) must be Tensor, not str

I had to add this line to the predict function to make it work:
`    return pred.start_logits, pred.end_logits, pred.attentions
`
I also had more errors with LayerConductance, but didn't debug this further. Reverting to version 3.5 of transformers resolved the issue, but it would be good to update the tutorial to be compatible with the latest transformers and also specify required / minimum versions if necessary.

2. For analyzing attributions for attention weights, for particularly this portion "head 10 learns strong relationship between what and include, heads 3, 8 between important and to and head 1 between to and what ...", should we instead say that these relationships are important to the prediction of the start token? It seems that there could be other relationships learned (e.g. others in the attention weights), but these attentions are most relevant for the particular decision, is that correct?

More generally, I'm curious here of how similar the attention weights are to the attributions when only looking at the specific row relevant to prediction (argmax start token). Other tokens from earlier layers still have some effect on the final prediction for the specific token through contribution to value vector, etc., but when isolating to the particular row, it was a bit difficult to compare with attention weights to see if there's any similarity.

3. nit: torch.norm is being deprecated, might be good to switch to torch.linalg.norm?

4. A few wording nits:
- nit: "need to computation of vector norms" - > "need to compute vector norms"
- nit: "Not let's transform" -> "Now let's transform"

File: website/tutorials.json [hunk 1]
```diff
@@ -25,7 +26,11 @@
     },
     {
       "id": "Bert_SQUAD_Interpret",
-      "title": "Interpreting question answering with BERT"
+      "title": "Interpreting question answering with BERT Part 1"
+    },
+    {
+      "id": "Bert_SQUAD_Interpret2",
+      "title": "Interpreting question answering with BERT Part 2"
```
[the remark this answers is not in the document]
Comment (NarineK) [reply website/tutorials.json:33]: Sounds good! I'll do it in a separate PR.

**Comment by NarineK** [comment]
> 1. head 10 learns strong relationship between

Thank you for the feedback @vivekmig!
1. Yeah, sorry about that I submitted the PR for the older Bert I'll push the changed for the Bert version 4.3+.
2. Yes, that's right, it is for start position prediction. I'll update the text.
     In terms of comparing the rows of attention matrices and attribution matrices. It is cumbersome I agree. Probably having a similarity metric to quantify it might be interesting.
3. Thank you for other nits regarding typos etc. I'll address them

**Review by bilalsal — APPROVED** [review]
All LGTM, approving to enable merging

**Pull request closed without merging**